test/run_test.sh

```diff
@@ -86,7 +90,8 @@ $MOVE_LOG_COMMAND || true
 # Injector commands for re-use
 # We can either test --simulate, and get partial data, or get full data
 # It's better to get full data
-INJECTOR_BASE_CMD="privcount inject --log events.txt"
+GZIP_CMD="gzip -c -d events2.txt.gz"
```

Comment:
I'm not sure we should compress the events file.
Even though we get 19x compression, git can't diff this file.
So every time we change the event format (for example, #178), we will be committing another copy of the file.
This only gives us ~20 changes to the file before we would have been better with an uncompressed file.